README.md

```diff
@@ -61,7 +61,7 @@ Install base dependencies:
 
 Requirements:
 
-- Ansible >= 2.4.3.0, 2.5.x is not currently supported for OCP installations
+- Ansible >= 2.4.3.0, 2.5.x+ are not currently supported for OCP installations
```

Comment:
Don't we want to change this? Perhaps with a note that it's currently only required for master / 3.11?
` - Ansible >= 2.6.0`

Reply:
Agreed.
`- Ansible >= 2.6.0`